src/Majordomo/MDPClientAsyncExample/MDPClientAsyncExample.csproj

```diff
@@ -1,80 +1,51 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{E6115527-4D2B-4196-8C89-8A8D44141E5D}</ProjectGuid>
+    <ProjectGuid>{BA3B66DE-85AB-44F0-890B-E598C777880D}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>MDPClientAsyncExample</RootNamespace>
-    <AssemblyName>MDPClientAsyncExample</AssemblyName>
+    <RootNamespace>MDPClientExample</RootNamespace>
+    <AssemblyName>MDPClientExample</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
This is at odds with the `TargetFrameworks` below. I think a lot of the contents of this file can be removed.

Reply:
done